app/helpers/application_helper/button/orchestration_template_make_orderable.rb

```diff
@@ -1,10 +1,6 @@
 class ApplicationHelper::Button::OrchestrationTemplateMakeOrderable < ApplicationHelper::Button::Basic
```

Comment:
app/helpers/application_helper/button/orchestration_template_view_in_catalog.rb
is not the same, has a different error message